content/docs/index.md

```diff
@@ -1,3 +1,7 @@
+---
+template: 'doc-home'
```

Comment:
it looks like we made `cards` general enough, do we need a separate template now?

Reply:
You're right, I expected to need to do something in another component but everything went fine with custom component. 
I'll remove this template frontmatter and the existing home template. We may need a unique template in the future, but we'll add it when that time comes.

Reply:
hey, any update to this? are we removing the template?

Reply:
I totally can, and quite quickly, but considering we don't know the exact direction this page is going I haven't committed a change to it yet.

Given my comment on the home issue, I think the best course of action may be to actually go all-in on the home template and make it a standalone component. This allows us to not be limited in design and get the docs home we want up quickly.

My current plan is that once I get the design I'll make the template that's a React component implementing that design, and once that makes the Docs page match what we want, we merge to master- this first step of the plan lets us get a great docs home out before the important DVC 1.0 events around the corner because the implementation is already here and React components are the easiest to port to any other method down the line.

After we have a page we want made in a simple React component template, we can later quite easily port the page content to something more dev/editor friendly that lets editors change the page in MD.

Reply:
@rogermparent I would be happy if we can get something simple and clean like this - https://cube.dev/docs . I don't think we need something too complicated/custom at this point. Does it help to make a decision? 

Also, we can take design ideas there (except icons of course :) ) . I'll ask designer after that to make us icons.